test/fuzz/OWNERS

```diff
@@ -0,0 +1,12 @@
+# See the OWNERS docs at https://go.k8s.io/owners
+
+options:
+  # make root approval non-recursive
+  no_parent_owners: true
+reviewers:
+  - sig-api-machinery-api-reviewers
+approvers:
+  - sig-api-machinery-api-approvers
+labels:
+  - area/test
+  - sig/testing
```

Comment:
Please don't label this sig/testing if you're excluding us from approvers.

Reply:
/hold 
this comment is not resolved?
https://github.com/kubernetes/kubernetes/pull/125709/files#r1655292544

there is still sig/testing label in the list but i don't see sig testing approvers.